source/Halibut.Tests/PollingTentacleDequeuesRequestsInOrderFixture.cs

```diff
@@ -21,23 +20,18 @@ public class PollingTentacleDequeuesRequestsInOrderFixture : BaseTest
         [LatestClientAndLatestServiceTestCases(testNetworkConditions: false, testListening: false)]
         public async Task QueuedUpRequestsShouldBeDequeuedInOrder(ClientAndServiceTestCase clientAndServiceTestCase)
         {
-            IPendingRequestQueue ?pendingRequestQueue = null;
+            var halibutTimeoutsAndLimits = new HalibutTimeoutsAndLimitsForTestsBuilder().Build();
+            halibutTimeoutsAndLimits.PollingQueueWaitTimeout = TimeSpan.FromSeconds(1);
```

Comment:
The queues respect this timeout passed in this class, so we can just use that instead of making our own queue.